src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -323,7 +325,19 @@ private Line2D CreateRegionConnectionLine(Vector2[] points, Color connectionColo
             Width = Constants.PATCH_REGION_CONNECTION_LINE_WIDTH,
         };
 
-        AddChild(link);
+        // This is a rather hacky way of rendering the connection lines below the nodes
```

Comment:
Could this maybe be worked around by having 2 intermediate level nodes, ones for holding patch nodes and one for holding the lines?